tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
I think mine comes from seeing a lot of `import thing` and then everywhere in code it was `thing.aces` or `thing.CoolClass.isit`, I'm more on the side of being explicit. `from thing import aces`, makes it easier to see what exactly is being imported and doesn't litter up your code and line length with module names.